docs/userguide/networking/get-started-overlay.md

```diff
@@ -91,7 +93,7 @@ that machine options that are needed by the `overlay` network driver.
 
 		$ docker-machine create \
 		-d virtualbox \
-		--swarm --swarm-image="swarm" --swarm-master \
+		--swarm --swarm-master \
```

Comment:
While you're at it, can you move `--swarm-master` to the next line; it's easier to read if each option is on its own line